lua/autorun/gamehints.lua

```diff
@@ -7,7 +7,7 @@ SwampHints["shop"] = "Press F3 to access the pointshop"
 SwampHints["disablehints"] = "To disable hints, open the scoreboard using tab and check the Display section"
 
 function SwampHint_InitVar()
```

Comment:
I'm pretty sure you can just do these like in https://github.com/swampservers/contrib/blob/master/gamemodes/cinema/gamemode/modules/theater/cl_commands.lua

Reply:
Well when it was accessing it as a var someone complained that it wasn't removing some of the hints.